tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -71,15 +74,19 @@ def read_results_from_file(self):
                     break
         return full_data
 
-    def analyze_results(self):
+    def analyze_results(self, tst):
```

Comment:
@Avilir add doc string is missing for this param.

Reply:
done